source/system/Entry.js

```diff
@@ -342,7 +360,9 @@ Entry.prototype._getWestley = function() {
 };
 
 Entry.Attributes = Object.freeze({
-    FacadeType:            "BC_ENTRY_FACADE_TYPE"
+    FacadeType:            "BC_ENTRY_FACADE_TYPE",
+    PassModifiedTime:      "passModifiedTime",
+    PassExpiry:            "passExpiry"
```

Comment:
Let's set these values to `BC_ENTRY_PASSMOD` and `BC_ENTRY_PASSEXP` to remain consistent.